### PR TITLE
Redesigned UI/UX for Main Website and Dashboard

### PR DESCRIPTION
**Pull request by TechGenius7777**
Redesigned UI/UX for Main Website and Dashboard. This has now been optimized for Mobile user also.

### CODE REVIEW

**Comment by Kakarot-0690** [comment]
looks pretty good

**Review by Kakarot-0690 — APPROVED** [review]

**Comment by TechGenius7777** [comment]
Thanks Kakarot!

**Comment by SudhanPlayz** [comment]
Change the colors of these 
image
image

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Change colors

**Comment by SudhanPlayz** [comment]
the link colors here too
image

File: views/index.html [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
+  width: 100%;
+  z-index: 100;
+  background: rgba( 255, 255, 255, 0.35 );
+box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
+backdrop-filter: blur( 9px );
+-webkit-backdrop-filter: blur( 9px );
+border: 1px solid rgba( 255, 255, 255, 0.18 );
+height: 10%">
+			<div class="container-fluid">
+				<a class="navbar-brand" href="#" style="color: white;">
```
Comment (SudhanPlayz) [inline views/index.html:36]:
```suggestion
				<a class="navbar-brand" href="#" style="color: white;
				    display: flex;
         	                    align-items: center;
		                    justify-content: center;">
```

Comment (SudhanPlayz) [reply views/index.html:36]: Also in `index.html` in navbar change Discord Music Bot to `<span style="margin-left: 10px;">Discord Music Bot</span>`

Comment (TechGenius7777) [reply views/index.html:36]: Umm Ok

**Comment by TechGenius7777** [comment]
> Change the colors of these image image

To which colour?

**Comment by SudhanPlayz** [comment]
> > Change the colors of these image image
> 
> To which colour?

Just choose any color which will be bright in the background

**Comment by joeyk710** [comment]
> > > Change the colors of these image image
> 
> > 
> 
> > To which colour?
> 
> 
> 
> Just choose any color which will be bright in the background 

Instead of saying change the color, TELL HIM what you're thinking. How bout you actually talk with him in discord.

**Comment by TechGenius7777** [comment]
I think extra changes would ruin the aesthetics. So can you please merge without any changes.

**Review by SudhanPlayz — APPROVED** [review]

**Pull request merged**